From bc92f601354c9336eff04aaa334c3993286accb9 Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Wed, 6 May 2020 07:50:46 -0500
Subject: [PATCH 27/31] arm64: dts: beacon-imx8mm: Clean up comments/dead code

There are some commented-out code and some unnecessary and/or
wrong comments.

This patch removes the dead code and unnecessary comments, and it
fixes comments that were incorrect.

There are no functional changes to this patch.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi | 6 +-----
 arch/arm64/boot/dts/freescale/beacon-imx8mm-kit.dts        | 2 +-
 arch/arm64/boot/dts/freescale/beacon-imx8mm-som.dtsi       | 2 +-
 3 files changed, 3 insertions(+), 7 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi b/arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi
index 4b1202ff4e28..f156b25aa167 100644
--- a/arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi
+++ b/arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: (GPL-2.0 OR MIT)
 /*
- * Copyright 2020 Beacon EmbeddedWorks
+ * Copyright 2020 Compass Electronics Group, LLC
  */
 
 / {
@@ -184,7 +184,6 @@
 		>;
 	};
 
-	/* Added for LogicPD Phoenix SOM*/
 	pinctrl_i2c4: i2c4grp {
 		fsl,pins = <
 			MX8MM_IOMUXC_I2C4_SCL_I2C4_SCL		0x400001c3
@@ -383,7 +382,6 @@
 			};
 		};
 	};
-
 };
 
 &micfil {
@@ -538,7 +536,6 @@
 	pinctrl-names = "default";
 	disable-over-current;
 	dr_mode="host";
-	/*gpio = <&pca6416_1 7 0>;*/ /* nRST */
 	status = "okay";
 };
 
@@ -546,7 +543,6 @@
 	reset-gpios = <&pca6416_1 7 GPIO_ACTIVE_HIGH>;
 };
 
-/*rfee  from uboot*/
 &usdhc2 {
 	pinctrl-names = "default", "state_100mhz", "state_200mhz";
 	pinctrl-0 = <&pinctrl_usdhc2>, <&pinctrl_usdhc2_gpio>;
diff --git a/arch/arm64/boot/dts/freescale/beacon-imx8mm-kit.dts b/arch/arm64/boot/dts/freescale/beacon-imx8mm-kit.dts
index f6d91309ba30..94ffd8fbf5b7 100644
--- a/arch/arm64/boot/dts/freescale/beacon-imx8mm-kit.dts
+++ b/arch/arm64/boot/dts/freescale/beacon-imx8mm-kit.dts
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: (GPL-2.0 OR MIT)
 /*
- * Copyright 2020 Beacon EmbeddedWorks
+ * Copyright 2020 Compass Electronics Group, LLC
  */
 
 /dts-v1/;
diff --git a/arch/arm64/boot/dts/freescale/beacon-imx8mm-som.dtsi b/arch/arm64/boot/dts/freescale/beacon-imx8mm-som.dtsi
index 583f572b000a..75cf1ef1e27b 100644
--- a/arch/arm64/boot/dts/freescale/beacon-imx8mm-som.dtsi
+++ b/arch/arm64/boot/dts/freescale/beacon-imx8mm-som.dtsi
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: (GPL-2.0 OR MIT)
 /*
- * Copyright 2020 Beacon EmbeddedWorks
+ * Copyright 2020 Compass Electronics Group, LLC
  */
 / {
 	usdhc1_pwrseq: usdhc1_pwrseq {
-- 
2.17.1

